src/installation/guides/chroot.md

```diff
@@ -0,0 +1,249 @@
+# Installation via chroot
+
+## Prepapre Filesystems
+
+These instructions can be followed on any running Linux system that has xbps
+installed, including an already working installation of Void or an official
+[live image](../live-images/prep.md). It is assumed that the host computer is
+connected to the Internet.
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) is available on live
+distributions for partitioning, but you may wish to use
+[fdisk(8)](https://man.voidlinux.org/fdisk.8),
```

Comment:
fdisk is also available on the live images.

Reply:
good catch!